theano/gof/tests/test_destroyhandler.py

```diff
@@ -43,6 +43,10 @@ def MyConstant(data):
 
 class MyOp(Op):
 
+    __props__ = ("nin", "name", "vmap", "dmap", "nout",
+                 "destroyhandler_tolerate_same",
+                 "destroyhandler_tolerate_aliased")
```

Comment:
For that one, given that lists and dictionaries are not hashable, and that no `__eq__` and `__hash__` were defined in the first place, it may be better not to define `__props__` at all.